addon/components/lottie.ts

```diff
@@ -66,12 +66,17 @@ export default class LottieComponent extends Component<LottieArgs> {
 
     this.args.onDataReady?.();
 
+    const { containerId } = this.args;
+    const container = containerId
+      ? (document.querySelector(`#${containerId}`) as Element)
+      : element;
+
     this.animation = lottie.loadAnimation({
       name: this.args.name,
       loop: this.args.loop,
       autoplay: this.autoplay,
       animationData,
-      container: document.querySelector(`#${this.args.containerId}`) || element,
+      container,
```

Comment:
```suggestion
      container: document.getElementById(this.args.containerId) || element,
```

I think we could simplify to this.

Reply:
This would defeat the purpose of this PR. What I'd like to do is avoid searching the DOM altogether if the `containerId` hasn't been passed as an argument.

Let's imagine this particular case:

* `containerId` hasn't been passed to the component
* `containerId` is thus `undefined`
* we will call `document.getElementById(undefined)`
* if the page has an element with `id` equal to `undefined`, this will return that element

I think this leads to unexpected behaviors, that's why my suggestion is to avoid searching the DOM.

Reply:
Gotcha. Makes sense.

Reply:
Thanks for reviewing @herzzanu <3